apps/gitness/package.json

```diff
@@ -46,6 +46,7 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.9.0",
+    "@git-diff-view/react": "^0.0.16",
```

Comment:
please raise a SECOPS ticket to review this package.

Reply:
it is already approved and present in views package as well